src/actionlist/action-list-item.scss

```
@@ -34,28 +34,42 @@
   }
 
   // only hover li without list as children
-  &:not(.ActionList-item--hasSubItem) {
+  &:not(.ActionList-item--hasSubItem),
+  // target contents of first child li if sub-item (li wraps item label + nested ul)
+  &.ActionList-item--hasSubItem > .ActionList-content {
     &:hover {
       cursor: pointer;
       background-color: var(--color-action-list-item-default-hover-bg);
+
+      // stylelint-disable-next-line selector-max-specificity
+      &:not(.ActionList-item--navActive) {
+        outline: $border-style $border-width var(--color-action-list-item-default-hover-border);
+        outline-offset: -$border-width;
+      }
     }
 
     &:active {
       background: var(--color-action-list-item-default-active-bg);
 
+      // stylelint-disable-next-line selector-max-specificity
+      &:not(.ActionList-item--navActive) {
+        outline: $border-style $border-width var(--color-action-list-item-default-active-border);
```

@langermank what do you think about using an `inset` `box-shadow` property for  this?

I think not supporting the rounded corners in Safari with `outline` is a major issue considering this is such a common component that would look visually broken.

I believe that the only other use case of `box-shadow` in an `ActionList-item` is in the 🐰 🥚 _delayed click_ functionality, but we can stack that shadow with an `inset` one on top there too.

```suggestion
        box-shadow: inset 0 0 0 $border-width var(--color-action-list-item-default-active-border);
```